README.md

```diff
@@ -1,12 +1,12 @@
 # LiMeApp
-[![Greenkeeper badge](https://badges.greenkeeper.io/libremesh/lime-app.svg)](https://greenkeeper.io/) [![Build Status](https://travis-ci.org/libremesh/lime-app.svg?branch=develop)](https://travis-ci.org/libremesh/lime-app) [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+[![Build Status](https://travis-ci.org/libremesh/lime-app.svg?branch=develop)](https://travis-ci.org/libremesh/lime-app) [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
 
 
-**Simple, lightweight and scalable PWA for diagnosis of Libremesh nodes**
+**Geek-free Web App for setup and mantainance of Libremesh nodes built on Preact**
 
 
 <p align="center"><br><br>
-    <img src="https://raw.githubusercontent.com/libremesh/lime-app/fd31c213/screenshot.gif" alt="Screenshot" />
+    <img height="480" src="docs/assets/screenshots.gif" alt="Screenshots" />
```

Comment:
maybe we can update this gif?
doesn't represent the current state of the app

Reply:
Hm... It does, are we both looking at the same gif? This is the one proposed: https://github.com/germanferrero/lime-app/blob/improve-docs/docs/assets/screenshots.gif
All the screens there are current ones. There are some missing of course